portality/bll/services/concurrency_prevention.py

```diff
@@ -0,0 +1,18 @@
+from portality.core import app
+import redis
+
+
+class ConcurrencyPreventionService():
+    def __init__(self):
+        self.rs = redis.Redis(host=app.config.get("REDIS_HOST"), port=app.config.get("REDIS_PORT"))
+
+    def checkConcurrency(self, key, id):
+        """
+        Checks whether concurrent request has been submitted
+        Returns true if clash is detected
+        """
+        value = self.rs.get(key)
+        return value is not None and value != id
+
+    def storeConcurrency(self, key, id, timeout=10):
```

Comment:
Should be the new config key `UR_CONCURRENCY_TIMEOUT` rather than hardcoded to 10s

Set this variable at instantiation so we can define timeout outside the class (since the timeout is UR-specific)